### PR TITLE
test-case: manual: check ASRC topology

### PR DESCRIPTION
**Pull request by ClarexZhou**
Signed-off-by: clarex <clarex.zhou@intel.com>

Add ASRC verify for manually check first

### CODE REVIEW

**Comment by marc-hb** [comment]
Can this not be automated more?

File: test-case/manual/check-asrc-tplg.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ASRC topology test
+ASRC topology test is manual test for now
+
+# Preconditions
+1. Rename ASRC tplg to normal tplg to use
+2. PulseAudio should be enabled
```
Comment (fredoh9) [inline test-case/manual/check-asrc-tplg.md:6]: based on below test steps, does pulse audio matter?

**Comment by fredoh9** [comment]
> Can this not be automated more?

Based on test steps, it is same case with eq test, which download eq tplg and over-write and run test case(s). We can automate this.

File: test-case/manual/check-asrc-tplg.md (hunk 1 above)
Comment (ClarexZhou) [reply test-case/manual/check-asrc-tplg.md:6]: pulseaudio isn't required

**Pull request merged**